src/main/kotlin/net/irisshaders/lilybot/extensions/moderation/TemporaryModeration.kt

```diff
@@ -104,17 +105,17 @@ class TemporaryModeration : Extension() {
 
 			check { anyGuild() }
 			check { hasPermission(Permission.ModerateMembers) }
+			check { configPresent() }
 
 			action {
-				val actionLogId = getConfigPrivateResponse("modActionLog") ?: return@action
-
+				val config = DatabaseHelper.getConfig(guild!!.id)!!
+				val actionLog = guild?.getChannel(config.modActionLog) as GuildMessageChannelBehavior
 				val userArg = arguments.userArgument
-				val actionLog = guild?.getChannel(actionLogId) as GuildMessageChannelBehavior
 
 				isBotOrModerator(userArg, "warn") ?: return@action
```

Comment:
I don't like this using string mapping, can we change that?

Reply:
No, that has nothing to do with the database and merely serves as a logging identifier